helm/fluent-logshipping-app/templates/priorityClass.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.fluentbit.customEnabled }}
```

Comment:
customEnabled? Can you maybe use something like 
```suggestion
{{- if .Values.fluentbit.priorityClass.create }}
```

Reply:
Sure, makes more sense